system/Commands/Utilities/Routes.php

```diff
@@ -101,9 +104,12 @@ public function run(array $params)
                     $sampleUri = $uriGenerator->get($route);
                     $filters   = $filterCollector->get($method, $sampleUri);
 
+                    $routeName = $collection->getRoutesOptions($route)['as'] ?? null;
```

Comment:
Shouldn't this properly be:
```suggestion
                    $routeName = $collection->getRoutesOptions($route)['as'] ?? '';
```

Reply:
In fact, when the `$routeName` is null, the true route name is the *Route* value (`$route`).

Since there is no explicit route name set, it shows blank, but I would like to change it if there is some better symbol or something.